src/index.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import {Router} from 'react-router-dom'
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import {Provider} from 'react-redux'
+import history from './utils/history'
+import store from './store'
+import './index.css';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <Provider store={store}>
+    <Router history={history}>
     <App />
-  </React.StrictMode>,
+  </Router> 
+  </Provider>, 
```

Comment:
Interested to know why we moved these, they're basically in the same place, only now it feels less clean in this file.

Reply:
Like I mentioned, just following FS/GH's lead from boilermaker. I'm fine with switching it if you'd prefer. 

Also I thought I already removed React.StrictMode, which shouldn't be there. I'll remove it and make another PR with that + Ayana's changes.